src/axolotl/cli/__init__.py

```diff
@@ -272,7 +272,7 @@ def do_inference_gradio(
             importlib.import_module("axolotl.prompters"), prompter
         )
     elif cfg.chat_template:
-        chat_template_str = get_chat_template(cfg.chat_template)
+        chat_template_str = get_chat_template(cfg.chat_template, tokenizer=tokenizer)
```

Comment:
Nice catch. While testing this, I got this error but figured it was due to my tokenizer chosen not having a template in its config.